apps/content/api/portal/translation_versions.py

```
@@ -0,0 +1,170 @@
+    "translations/{translation_slug}/versions/",
+    response={
+        201: TranslationVersionListOut,
+        400: NinjaErrorResponse[Literal["asset_id_mismatch"], Literal[None]],
+        404: NinjaErrorResponse[Literal["translation_not_found"], Literal[None]],
+    },
+)
+def create_translation_version(
+    request: Request,
+    translation_slug: str,
+    data: Form[TranslationVersionCreateIn],
+    file: UploadedFile = File(...),
```

Ruff (0.15.10)

[warning] Do not perform function call `File` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)